doc/api/child_process.markdown

```diff
@@ -37,7 +37,7 @@ Process instance.
   was killed by the parent.
 
 This event is emitted when the stdio streams of a child process have all
-terminated.  This is distinct from 'exit', since multiple processes
+terminated. This is distinct from 'exit', since multiple processes
```

Comment:
This double spacing is not significant in rendered markdown and we use it in a few other places. Maybe leave out this change?